examples/hello-compute/main.rs

```diff
@@ -90,9 +90,9 @@ fn main() {
 
     queue.submit(&[encoder.finish()]);
 
-    staging_buffer.map_read_async(0, size, |result: wgpu::BufferMapAsyncResult<&[u32]>| {
-        if let Ok(mapping) = result {
-            println!("Times: {:?}", mapping.data);
-        }
-    });
+    let result = futures::executor::block_on(staging_buffer.map_read::<u32>(0u64, size));
```

Comment:
We'll need to find a good way for examples to switch between executors on native vs. web, and make sure other executors work too (for example, we can't use `block_on` on the web)

Reply:
I was considering putting the entire example in an `async` function, and then we can use `await` throughout, and make main just an executor wrapper.